lib/hsa/mcwamp_hsa.cpp

```diff
@@ -4847,7 +4844,7 @@ static std::string fenceToString(int fenceBits)
 
 
 bool HSABarrier::isReady() override {
-    bool ready = (hsa_signal_load_acquire(signal) == 0);
+    bool ready = (hsa_signal_load_acquire(_signal) == 0);
```

Comment:
Unless I am mistaken, this is deprecated, with `hsa_signal_load_scacquire` being the current / preferred form.